Recently the license file was removed from the setuptools distribution.

<https://github.com/jaraco/skeleton/pull/171>

Aside from the obvious legal issue with no longer distributing the license,
this obviously causes us problems on MassOS, where we want to ensure we install
the license file(s) for every built-in package.

This patch simply restores the file, so as to ensure we can install it in the
way we expect to.

diff '--color=auto' -aruN a/LICENSE b/LICENSE
--- a/LICENSE	1970-01-01 01:00:00.000000000 +0100
+++ b/LICENSE	2025-05-05 12:43:01.757126224 +0100
@@ -0,0 +1,17 @@
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to
+deal in the Software without restriction, including without limitation the
+rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+sell copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+IN THE SOFTWARE.
